tests/hikari/test_guilds.py

```diff
@@ -161,6 +162,9 @@ def test_mention_property_when_no_nickname(self, obj):
         obj.nickname = None
         assert obj.mention == "<@123>"
 
+    def test_top_role(self, obj):
+        assert obj.top_role.guild_id == "Owner"
```

Comment:
This is not the right way of testing this. You will have to mock `app.cache.get_roles_view_for_guild`, make it return a dict with the values you expect, make sure it gets called with the right guild_id and that the role return is the correct one. You will also have to write tests (and modify the code you wrote) for when an empty role view is returned (missing intents so we didnt cache it or he is using a stateless bot).